GenderGapsInCollegeDegrees.md

```diff
@@ -0,0 +1,160 @@
+
+## Clean up the chart
+1. Let's explore the gender gap in STEM fields, such as science, technology, engineering, and mathematics.
+2. First off, start with the historical percentage of Biology degrees for both genders.
+
+
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+#Read data
+women_degrees = pd.read_csv('percent-bachelors-degrees-women-usa.csv')
+#Visualize the gender gap of Biology degree
+fig, ax = plt.subplots()
+ax.plot(women_degrees['Year'], women_degrees['Biology'], c='red', label='Women')
+ax.plot(women_degrees['Year'], 100-women_degrees['Biology'], c='blue', label='Men')
+#Disable tick marks
+ax.tick_params(bottom="off", top="off", left="off", right="off")
+#Erase spines
+for key,spine in ax.spines.items():
+    spine.set_visible(False)
+#Display the chart
+ax.legend(loc='upper right')
+plt.show()
+```
+
+
+![png](output_1_0.png)
```

Comment:
It may be useful to have some analysis / speculation after this chart.  Why did the percentage of women increase so dramatically from 1995-2005?  Why has the percentage gone down slightly recently?  In cases like this, some google searching can help.  Here are some articles that *may* be useful:

* [ASU](https://asunow.asu.edu/content/study-shows-gender-gaps-remain-introductory-science-classrooms)
* [NYT](http://www.nytimes.com/2011/11/06/education/edlife/where-the-women-are-biology.html)

It could also be useful to look at the total number of women and men that are majoring in biology, not just the percentages.  Has the total number gone down recently?  Has the number of men remained constant while the number of women has decreased, increasing the male percentage?  

More data will help you tease apart why the percentages are changing.